src/freeclimb.Test/Model/TranscribeUtteranceTests.cs

```diff
@@ -0,0 +1,282 @@
+ *
+ * The version of the OpenAPI document: 1.0.0
+ * Contact: support@freeclimb.com
+ * Generated by: https://github.com/openapitools/openapi-generator.git
+ */
+
+
+
+
+using Xunit;
+
+using System;
+using System.Linq;
+using System.IO;
+using System.Collections.Generic;
+using System.Text;
+using freeclimb.Api;
+using freeclimb.Model;
+using freeclimb.Client;
+using System.Reflection;
+using Newtonsoft.Json;
+using System.Globalization;
+using freeclimb.Enums;
+
+
+namespace freeclimb.Test.Model
+{
+    /// <summary>
+    ///  Class for testing TranscribeUtterance
+    /// </summary>
+    /// <remarks>
+    /// This file is automatically generated by OpenAPI Generator (https://openapi-generator.tech).
+    /// Please update the test case below to test the model.
+    /// </remarks>
+    public class TranscribeUtteranceTests
+    {
+        private TranscribeUtterance instance;
+
+        public TranscribeUtteranceTests()
+        {
+            instance = new TranscribeUtterance("https://a.com");
+        }
+
+        /// <summary>
+        /// Test an instance of TranscribeUtterance
+        /// </summary>
+        [Fact]
+        public void TranscribeUtteranceInstanceTest()
+        {
+            Assert.IsType<TranscribeUtterance>(instance);
+        }
+
+
+        /// <summary>
+        /// Test the property 'ActionUrl'
+        /// </summary>
+        [Fact]
+        public void ActionUrlTest()
+        {
+            instance.ActionUrl = "TEST_STRING";
+            Assert.Equal("TEST_STRING", instance.ActionUrl);
+
+        }
+        /// <summary>
+        /// Test the property 'PlayBeep'
+        /// </summary>
+        [Fact]
+        public void PlayBeepTest()
+        {
+            instance.PlayBeep = false;
+            Assert.Equal(false, instance.PlayBeep);
+
+        }
+        /// <summary>
+        /// Test the property 'Record'
+        /// </summary>
+        [Fact]
+        public void RecordTest()
+        {
+            instance.Record = new TranscribeUtteranceRecord();
+            Assert.IsAssignableFrom<TranscribeUtteranceRecord>(instance.Record);
+        }
+        /// <summary>
+        /// Test the property 'PrivacyForLogging'
+        /// </summary>
+        [Fact]
+        public void PrivacyForLoggingTest()
+        {
+            instance.PrivacyForLogging = false;
+            Assert.Equal(false, instance.PrivacyForLogging);
+
+        }
+        /// <summary>
+        /// Test the property 'PrivacyForRecording'
+        /// </summary>
+        [Fact]
+        public void PrivacyForRecordingTest()
+        {
+            instance.PrivacyForRecording = false;
+            Assert.Equal(false, instance.PrivacyForRecording);
+
+        }
+        /// <summary>
+        /// Test the property 'Prompts'
+        /// </summary>
+        [Fact]
+        public void PromptsTest()
+        {
+            //object[] testArray = {"ElementOne", "ElementTwo", "ElementThree"};
+            //instance.Prompts = testArray;
```

Comment:
Looks like test Array is now being used in the C# SDK. We'll need to uncomment the isArray flag in the `model_test` mustache file.